app/src/main/res/drawable/ic_warning_black.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+  <path
+      android:fillColor="#FF000000"
```

Comment:
I'm still counting 4 spaces indentation.

Reply:
Oops, VI somehow expanded tabs to 4 spaces instead of 2 spaces before push :/
Fixed now.

Reply:
@ashutoshbsathe No problem. @mhutti1 will review it soon.